plugins/git/git.plugin.zsh

```diff
@@ -103,20 +103,20 @@ function gccd() {
 compdef _git gccd=git-clone
 
 alias gcl='git clone --recurse-submodules'
-alias gclean='git clean -id'
-alias gpristine='git reset --hard && git clean -dffx'
+alias gclean='git clean --interactive -d'
+alias gpristine='git reset --hard && git clean --force -dx'
 alias gcm='git checkout $(git_main_branch)'
 alias gcd='git checkout $(git_develop_branch)'
-alias gcmsg='git commit -m'
+alias gcmsg='git commit --message'
 alias gco='git checkout'
 alias gcor='git checkout --recurse-submodules'
-alias gcount='git shortlog -sn'
+alias gcount='git shortlog --summary --numbered'
 alias gcp='git cherry-pick'
 alias gcpa='git cherry-pick --abort'
 alias gcpc='git cherry-pick --continue'
 alias gcs='git commit -S'
-alias gcss='git commit -S -s'
-alias gcssm='git commit -S -s -m'
+alias gcss='git commit -S --signoff'
+alias gcssm='git commit -S --signoff --message'
```

Comment:
`-S` should be replaceable with `--gpg-sign` according to the [git documentation](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--Sltkeyidgt)

Reply:
OK I finally got the chance to fix this up; I am pretty sure there are no stragglers left but looking forward to your review. Thanks for your patience!